### PR TITLE
[Merged by Bors] - chore: more unused variables

### PR DESCRIPTION
**Pull request by adomani**
#17715

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks! Just two very minor comments.
maintainer merge

**Comment by sgouezel** [comment]
bors d+
Thanks!

File: Mathlib/LinearAlgebra/BilinearForm/Properties.lean [hunk 1]
```diff
@@ -116,7 +115,6 @@ theorem isSymm_zero : (0 : BilinForm R M).IsSymm := fun _ _ => rfl
 theorem isSymm_neg {B : BilinForm R₁ M₁} : (-B).IsSymm ↔ B.IsSymm :=
   ⟨fun h => neg_neg B ▸ h.neg, IsSymm.neg⟩
 
-variable (R₂) in
```
[the remark this answers is not in the document]
Comment (adomani) [reply Mathlib/LinearAlgebra/BilinearForm/Properties.lean:119]: R2 is not used in the lemma, although R is and is implicit.  The lemma is only used once, only in this file and when it is used it is irrelevant whether the argument `R` is implicit or not.  I am just going to remove the unused variable and if someone wants to make explicit another variable, then they can do so in a later PR.

**Comment by adomani** [comment]
Thank you for your comments!

bors merge

**Pull request closed without merging**